CoreWebServer/src/au/gov/asd/tac/constellation/webserver/restapi/RestService.java

```diff
@@ -38,9 +38,56 @@
  * <p>
  * All services are accessed via RestServiceServlet; see that for more details.
  *
+ * This also stores status codes.
+ * Top section copied from public interface HttpServletResponse.
+ * 
  * @author algol
  */
 public abstract class RestService {
+    public static final int SC_CONTINUE = 100;
```

Comment:
Can you please use the `HttpURLConnection` class instead of creating these constants?

Reply:
422 is not there. Shall I leave only that in the new class then?

Reply:
Yep @Auriga2. Sorry I didn't realise 422 was not there.